corehq/apps/hqwebapp/static/cloudcare/less/formplayer-common/address.less

```diff
@@ -0,0 +1,8 @@
+// Overwrite mapboxgl geocoder library styling.
+.mapboxgl-ctrl-geocoder {
+  div& {
```

Comment:
What does the trailing `&` do?

Reply:
I just got it from this [answer](https://stackoverflow.com/a/14170471), I'm not familiar with less and thought this was standard. How do we do it? 
I wanted to target a class and element to be specific so we override the class alone settings.

Reply:
I see, it looks like the `&` syntax is intended as a shortcut if you have a bunch of styles that apply to a class, but you want slight variations depending on what type of element is using the class. Since you only have one variant, you don't need this, it's more typical to use the expression that this will evaluate to, `div.mapboxgl-ctrl-geocoder`. And then, since divs are such generic elements, we typically leave them off. So you can just delete the `div&` line altogether.

Reply:
I think when I didn't have the element, their css file took precedence so I made it more specific. Is there another way to force this?

Reply:
You could make it more specific by adding some other container selector, like `.widget .mapboxgl-ctrl-geocoder` or `#web-forms .mapboxgl-ctrl-geocoder`. So it's the same technical effect, just with a style that's more common in our code base.